Sources/SectionLabel.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  SectionLabel.swift
+//  Demo
+//
+//  Created by Marijn Schilling on 06/06/16.
```

Comment:
Please remove the attribution header, usually classes are modified by more than one dev, and it's hard to keep it updated.